src/NServiceBus.Metrics.ServiceControl/ServiceControlReporting/EndpointMetadata.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace NServiceBus.Metrics.ServiceControl.ServiceControlReporting
+{
+    class EndpointMetadata
+    {
+        readonly string localAddress;
+
+        public EndpointMetadata(string localAddress)
+        {
+            this.localAddress = localAddress;
+        }
+
+        public string ToJson()
+        {
+            return SimpleJson.SerializeObject(new
+            {
+                Version = 3,
```

Comment:
Why `3`. Put a comment in here please.

Reply:
This is the plugin version. Should this use reflection to retrieve the value? 

This version is being used to figure out what's likely to be in the queue names. Should this be Core version? Is that available somewhere?